core.js

```diff
@@ -1960,7 +1836,21 @@ exokit.setNativeBindingsModule = nativeBindingsModule => {
   };
 
   const {nativeAudio} = bindings;
-  AudioContext = nativeAudio.AudioContext;
+  AudioContext = class AudioContext extends nativeAudio.AudioContext {
+    /**
+     * Wrap AudioContext.DecodeAudioDataSync binding with promises and callbacks.
+     */
+    decodeAudioData(arrayBuffer, successCallback, errorCallback) {
+      try {
+        let audioBuffer = this._decodeAudioDataSync(arrayBuffer);
+        if (successCallback) { successCallback(audioBuffer); }
```

Comment:
I think this should be called async (`process.nextTick` or `setImmediate`) for consistency. There is probably user code out there that assumes that's the case.